awssql/utils/rds_utils.go

```diff
@@ -58,6 +58,10 @@ var (
 	IP_V6_COMPRESSED_REGEXP = regexp.MustCompile("^(([0-9A-Fa-f]{1,4}(:[0-9A-Fa-f]{1,4}){0,5})?)" +
 		"::(([0-9A-Fa-f]{1,4}(:[0-9A-Fa-f]{1,4}){0,5})?)$")
 
+	BG_OLD_HOST_PATTERN     = regexp.MustCompile("(?i).*(?P<prefix>-old1\\.)...*") //nolint:all
+	BG_GREEN_HOSTID_PATTERN = regexp.MustCompile("(?i)(.*)-green-[0-9a-z]{6}")
+	BG_GREEN_HOST_PATTERN   = regexp.MustCompile("(?i).*(?P<prefix>-green-[0-9a-z]{6})...*")
```

Comment:
`BG_OLD_HOST_PATTERN` and `BG_GREEN_HOST_PATTERN` seem to have an extra `.` compared to the one in [jdbc](https://github.com/aws/aws-advanced-jdbc-wrapper/blob/55c9022f97af83b256b30acc73887cd968305dea/wrapper/src/main/java/software/amazon/jdbc/util/RdsUtils.java#L164C3-L177C37). Just wanted to verify if this was correct?